docs/achtergronddocumentatie/artefacten/9339.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: page-with-side-nav
+title: B9339 - De relatie *werkte voor* wordt niet geïmplementeerd
+date: 02-10-2023
+---
+
+# B9339 - De relatie *werkte voor* wordt niet geïmplementeerd
+
+## Besluit
+
+De relatie *werkte voor* wordt niet geïmplementeerd en dus niet opgenomen in het Technisch Uitwisselingsmodel. 
```

Comment:
Waarom? Geldt dit ook voor de 'vertegenwoordigde'-relatie van partij naar partij?

Reply:
Gaarne bespreken. Dit was mijn conclusie, maar blijkbaar geen gezamenlijke conclusie.

Reply:
Toelihcting door Ivo n.a.v. uitgewerkte casus in basisterminologie. De "werkte voor" wordt wel geïmplementeerd en is echt "werkte voor op het moment van een klantcontact" en heeft geen herbruikwaarde.

Reply:
artefact 9339.md en de verwijzing daarnaar verwijderd.